build/pipeline-uiautomation.yaml

```diff
@@ -121,7 +121,7 @@ stages:
       inputs:
         restoreSolution: LibsMacOS.sln
 
-    - task: XamariniOS@2
+    - task: XamariniOS@1
```

Comment:
What does this do?

Reply:
was trying to fix the yaml build issues. found it in a xamarin forum but it didn't work.